src/modules/aws_acm/entity/certificate.ts

```diff
@@ -13,64 +17,128 @@ export enum certificateStatusEnum {
   VALIDATION_TIMED_OUT = 'VALIDATION_TIMED_OUT',
 }
 
+/**
+ * @enum
+ * Specifies if the certificate has been generated by AWS or has been imported
+ */
+
 export enum certificateTypeEnum {
   AMAZON_ISSUED = 'AMAZON_ISSUED',
   IMPORTED = 'IMPORTED',
   // TODO: add private certs support
   // PRIVATE = "PRIVATE",
 }
 
+/**
+ * @enum
+ * Specifies if the certificate is available to be renewed or not
+ */
 export enum certificateRenewalEligibilityEnum {
   ELIGIBLE = 'ELIGIBLE',
   INELIGIBLE = 'INELIGIBLE',
 }
 
+/**
+ * Table to query for all AWS certificates in the system, managed by AWS ACM.
+ * AWS Certificate Manager (ACM) handles the complexity of creating, storing, and renewing public
+ * and private SSL/TLS X.509 certificates and keys that protect your AWS websites and applications.
+ *
+ * Certificates can be read and deleted, but not created or modified. Instead certificates can be
+ * created or imported by using the following postgres functions:
```

Comment:
sounds a bit weird to me that we are saying that Certificates cannot be created or modified but immediately after we are saying they actually can be created using the Postgres functions.